cosmos/providers/dbt/core/operators.py

```diff
@@ -215,14 +233,60 @@ def run_command(self, cmd, env):
                 raise AirflowException(
                     f"The project_dir {self.project_dir} must be a directory"
                 )
+
+        # routing the dbt project to a tmp dir for r/w operations
+        target_dir = f"/tmp/dbt/{os.path.basename(self.project_dir)}"
+
+        changes = True  # set changes to true by default
+        if os.path.exists(target_dir):
```

Comment:
So if I'm reading this correctly this will check for changes and copy over only when the actual task is run? Would it make more sense to do it every time the DAG is parsed? Especially if these models are showing up in the airflow UI I wouldn't want people to change their models and then not have that reflected

Reply:
Well, the users are going to make changes to their models in `./dags/dbt/` and it will be reflected on the UI - then when they run the task, it will be copied into `/tmp`